addons/link_tracker/models/link_tracker.py

```diff
@@ -84,6 +84,14 @@ def _compute_code(self):
             record = self.env['link.tracker.code'].search([('link_id', '=', tracker.id)], limit=1, order='id DESC')
             tracker.code = record.code
 
+    def _inverse_code(self):
```

Comment:
Would force ensure_one (as code is unique, makes no sense to be multi), it will simplify the code